modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -7,7 +7,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	disttypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
-	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
```

Comment:
I am looking into reverting the uses of `v1` but I face the following problems:

- Both `GovKeeper.SetProposal` and `GovKeeper.ActivateVotingPeriod` accept now only `v1` proposal arguments.
- In `testing/simapp/app.go`, the line `paramsKeeper.Subspace(govtypes.ModuleName).WithKeyTable(govv1.ParamKeyTable())` uses also `v1` because `ParamKeyTable` is not available in `v1beta1` anymore.

Any ideas?

Reply:
Got it, lets just leave it then. No need to spend engineering resources downgrading to an older version if the downgrade isn't trivial